content/latest/api/ysql/commands/dml_insert.md

```diff
@@ -49,17 +69,32 @@ Where
 - `column_names` is a comma-separated list of columns names (identifiers).
 
 ## Semantics
+
+The bare INSERT variant is straightforward.
+
  - An error is raised if the specified table does not exist. 
- - Each of primary key columns must have a non-null value.
+ - Each of the primary key columns must have a non-null value.
 
 ### `VALUES` Clause
  - Each of the values list must have the same length as the columns list.
  - Each value must be convertible to its corresponding (by position) column type.
  - Each value literal can be an expression.
 
+The INSERT ON CONFLICT variant needs some explanation. The target table must have
```

Comment:
Replace "The INSERT ON CONFLICT variant needs some explanation. "  with a title

### `ON CONFLICT` Clause